tests/integrations/tests/test_langchain.py

```
@@ -1384,6 +1382,104 @@ def test_26_thinking_bedrock(self, test_config, provider, model):
             else:
                 raise
 
+    # =========================================================================
+    # TOKEN COUNTING TEST CASES - get_num_tokens_from_messages
+    # =========================================================================
+
+    @pytest.mark.parametrize("provider,model", get_cross_provider_params_for_scenario("count_tokens"))
+    def test_26_get_num_tokens_simple_text(self, test_config, provider, model):
```

Ruff (0.14.10)

Unused method argument: `test_config`

(ARG002)